src/audio/pipeline.c

```diff
@@ -222,7 +222,8 @@ int pipeline_complete(struct pipeline *p, struct comp_dev *source,
 		.comp_data = &data,
 	};
 
-	pipe_info(p, "pipeline complete");
+	pipe_info(p, "pipeline complete, clock freq= %dkHz",
```

Comment:
Recent log messages just insert single space between name and value, no extra '='.

Reply:
updated